continuous_integration/travis/install.sh

```diff
@@ -18,7 +18,8 @@ if [ "$DISTRIB" == "conda" ]; then
     # force no mkl because mayavi requires old version of numpy
     # which then crashes with pandas and seaborn
     conda create --yes -n testenv python=$PYTHON_VERSION pip nomkl numpy\
-        setuptools matplotlib pillow pytest pytest-cov coverage seaborn
+        setuptools matplotlib pillow pytest pytest-cov coverage seaborn\
+        memory_profiler
```

Comment:
for example you could add to the `DISTRIB == ubnutu` case below:
```
pip uninstall memory_profiler
```
to hit the `except` lines you have now added

Reply:
Done, let's see what the CI says...